org.geogebra.GeoGebra.json

```diff
@@ -0,0 +1,44 @@
+{
+    "app-id": "org.geogebra.GeoGebra",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "1.6",
+    "sdk": "org.freedesktop.Sdk",
+    "command": "geogebra",
+    "finish-args": [
+        "--share=ipc",
+        "--share=network",
+        "--socket=x11",
+        "--socket=wayland",
+        "--filesystem=home"
+    ],
+    "modules": [
+        {
+            "name": "geogebra",
+            "buildsystem": "simple",
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://download.geogebra.org/installers/5.0/GeoGebra-Linux-Portable-5-0-367-0.tar.bz2",
```

Comment:
@TingPing I have pushed a new commit that should solve this and the other things marked in your review.
I couldn't find tarballs to download the specific versions of the JRE, so I added a script to remove the unwanted one with a `only-arches.